tests/components/myuplink/test_init.py

```diff
@@ -214,3 +215,39 @@ async def test_device_remove_devices(
         old_device_entry.id, mock_config_entry.entry_id
     )
     assert response["success"]
+
+
+@pytest.mark.parametrize(
+    "load_systems_file",
+    [load_fixture("systems-multi.json", DOMAIN)],
+)
+@pytest.mark.parametrize(
+    ("load_device_file", "device_id"),
+    [
+        (
+            load_fixture("device-alfred.json", DOMAIN),
+            "alfred-r-1234-20240201-123456-aa-bb-cc-dd-ee-ff",
+        ),
+        (
+            load_fixture("device-batman.json", DOMAIN),
+            "batman-r-1234-20240201-123456-aa-bb-cc-dd-ee-ff",
+        ),
+        (
+            load_fixture("device-robin.json", DOMAIN),
+            "robin-r-1234-20240201-123456-aa-bb-cc-dd-ee-ff",
+        ),
+    ],
+)
```

Comment:
2 ways to fix it:
1. Have the test fixture take the file name instead and do the fetching of the data in the fixture

2.
```suggestion
@pytest.mark.parametrize(
    ("load_device_file", "device_id"),
    [
        (
            load_fixture("device-alfred.json", DOMAIN),
            "alfred-r-1234-20240201-123456-aa-bb-cc-dd-ee-ff",
        ),
        (
            load_fixture("device-batman.json", DOMAIN),
            "batman-r-1234-20240201-123456-aa-bb-cc-dd-ee-ff",
        ),
        (
            load_fixture("device-robin.json", DOMAIN),
            "robin-r-1234-20240201-123456-aa-bb-cc-dd-ee-ff",
        ),
    ],
    ids=["..", "..", "..", ".."]
)
```

Reply:
First one would be way nicer btw

Reply:
Thank you for pushing me into a new lane...
Tried No 1 already. It does not work since the fixture is cached so same data is used for each iteration. Experimenting with scope did not change anything.
However, no 2 works fine and can also remove unnecessary name string coming from first parametrize statement.